image/util.go

```diff
@@ -42,6 +42,16 @@ func GetImageLayerDirs(image *v1.Image) (res []string, err error) {
 	return
 }
 
+//get docker image layer hash path
```

Comment:
this func is called outside pkg image. try use interface to provide service for outside pkg, or make it a local func for other pkg.